# trajectory Optimizer
mini_T:                   0.05
wei_time_:                10.0

kmax:                     1.665 
cons_eps_:                0.1



vmax:                     1.0
latAccmax:                10.0
lonAccmax:                5.0
accRatemax:               1.5
phimax: 0.785 #0.34

traj_res:                 6
past:                     7
rho:                      1.0
rho_max:                  1000.0
gamma:                    1.0
mem_size:                 256
g_epsilon:                1.0e-3
min_step:                 1.0e-32
delta:                    1.0e-6
max_iterations:           20000
amlMaxIter:               100

penaWei:                  1000.0
esdfWei:                  0.1

pieceTime:                0.7
miniS:                    0.1
# safeMargin:               0.25 #0.15+0.05
# conpts:                   [[0.15, 0.0], [0.3, -0.00]]
safeMargin:               0.2 #0.15+0.05
conpts:                   [[0.31, 0.29], [0.31, -0.29], [-0.31, -0.29], [-0.31, 0.29]]

start_vec:                [-9.5, 8, 0, 0.0]  # px py yaw v
end_vec:                  [ 9, -8, 0, 0.0]
FIRI_bound:               3.0

front_end_vis:            true
once_vis_one:             true
once_duration_ms:         -1
once_optimization:        true
skip_iter_vis:            100
skip_iter_duration_ms:    100
individual_vis:           false

#map
mapRes:                   0.2
mapX:                     25.0
mapY:                     25.0
expandSize:               1
wheel_base:               0.55
horizon_:                 50.0
#res res res
yaw_resolution_:          7
steer_res:                0.5
allocate_num_:            1000000
lambda_heu_:              1.0


#0:48134 1:21376 2:19494 3:5794 4:4041 5:3064
check_num_:               3
step_arc:                 1.5
checkl:                   0.1

max_seach_time:           2000.0
non_siguav:               0.0
backW:                    0.5
isdebug:                  false
isfixGear:                false
isVis:                    false
enable_shot:              true